src/styles/blocks/section.scss

```diff
@@ -0,0 +1,186 @@
+.section {
+  padding-top: 40px;
+
+  @include mobile-horizontal-paddings;
+
+  @include onTablet {
+    padding-top: 70px;
+    padding-left: 35px;
+    padding-right: 35px;
+  }
+
+  @include onDesctop {
+    padding-top: 90px;
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &--svg {
+      width: 50px;
+      height: 40px;
+    }
+  }
+
+  &__article {
+    min-width: 260px;
+    padding-top: 40px;
+
+    @include mobile-horizontal-paddings;
+
+    @include onTablet {
+      padding-top: 50px;
+      padding-left: 0;
+      padding-right: 0;
+    }
+
+    &--padding-top--none {
+      padding-top: 0;
+    }
+  }
+
+  &__title {
+    margin-bottom: 20px;
+    text-align: left;
+
+    @include onTablet {
+      font-size: 48px;
+      line-height: 62px;
+    }
+
+    @include onDesctop {
+      font-size: 62px;
+      line-height: 78px;
+    }
+
+    &-small {
+      margin-bottom: 20px;
+
+      @include onDesctop {
+        margin-bottom: 24px;
+      }
+
+      &--margin-b52 {
+        @include onDesctop {
+          margin-bottom: 52px;
```

Comment:
Why. do you use such a nesting here `&-`?

You could nest modifiers, or elements, but there you have non of them.

Watch more about naming convention in BEM [in this video](https://mate.academy/learn/css-advanced/bem#/video/920)
Watch closely when you use '-' in naming.